packages/framework/assets/js/admin/registerAdmin.js

```diff
@@ -8,9 +8,13 @@ import './validation/form';
 
 import Register from '../common/utils/Register';
 
-export default function registerAdmin () {
+export default function registerAdmin (afterRegistrationCallback = null) {
```

Comment:
Was this change necessary?

Reply:
It wasn't, but it occurred to me that it could be used by somebody because we have the option to influence admin only with registerCallbacks and priorities now.